suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -201,8 +201,22 @@ const getCardanoSupportedAccountTypesThunk = createThunk(
             return undefined;
         }
         const availableCardanoDerivationsResponse = await requestDeviceAccess({
-            deviceCallback: () =>
-                dispatch(getAvailableCardanoDerivationsThunk({ deviceState, device })).unwrap(),
+            deviceCallback: () => {
+                // calling method like getFeatures will make connect useCardanoDerivation set to true with use other calls
+                // using cardanoConnectPatch otherwise getting Cardano derivation might fail (if useCardanoDerivation is false)
+                TrezorConnect.getFeatures({
```

Comment:
why there is no await?

Reply:
good point. I’ll add it but it works anyway

Reply:
2e9e127b4e